addons/mail/static/src/js/utils.js

```diff
@@ -173,7 +173,7 @@ function inline(node, transform_children) {
 // Parses text to find email: Tagada <address@mail.fr> -> [Tagada, address@mail.fr] or False
 function parseEmail(text) {
     if (text) {
-        var result = text.match(/(.*)<(.*@.*)>/);
+        var result = text.match(/[\s"]*([^"]*)[^<]*<(.*@.*)>/);
```

Comment:
What is this change about ? Shoudln't it be explained in commit message ?

Reply:
You’re right, I didn’t reminded the exact reason so a comment in the commit is useful. I have added: 
Technical note: the regexp in the mail util parseEmail method has been slightly
modified because it was returning in some cases the name surrounded by quotes 
(ex.: "newlead3@ex.com" <newlead3@ex.com> --> name: "newlead3@ex.com"). This is
no longer the case with this change. That change was needed for comparing the 
name and the email to avoid to display the same thing twice (when the email and
the name are the same). Note that the method is only used once, where we have 
done the change.

Reply:
The best solution would be to never use regexp, but maybe because I never understood them :D .